mscore/inspector/inspector_fret.ui

```diff
@@ -237,11 +237,7 @@
        </widget>
       </item>
       <item row="6" column="2">
-       <widget class="Ms::ResetButton" name="resetShowNut" native="true">
-        <property name="accessibleName">
-         <string>Reset show nut</string>
```

Comment:
why remove the accessible name for this?

Reply:
Good question: because no other reset button in any inspector ui has them anymore.

Reply:
OK, fair enough. The latter option would be ideal but I can see that that's not something for this PR :)

Reply:
Yes, separate PR or separate commit to this here